### PR TITLE
Project added

### PR DESCRIPTION
**Pull request by JatinRanka**

### CODE REVIEW

**Comment by Akshay-Laxmipathi** [comment]
Delete playlist button is not functional

File: src/views/VideoPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const VideoPage = () => {
+	const { videoId } = useParams();
+	const {
+		state: { videosList }
+	} = useVideosContext();
+
+	const videoDetails = videosList.find((video) => video._id === videoId);
+
+	return <VideoPageComponent videoDetails={videoDetails} videosList={videosList} />;
```
Comment (Ashishgupta08) [inline src/views/VideoPage/index.jsx:14]: Limit similar videos up to 5 or 6

File: src/components/SaveVideoModal/index.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+					})}
+				</div>
+				<hr />
+
+				<input
+					placeholder="Add playlist name"
+					value={newPlaylistName}
+					onChange={(event) => setNewPlaylistName(event.target.value)}
+				/>
+				<button onClick={handleAddPlaylistAndSave}>Save</button>
```
Comment (Ashishgupta08) [inline src/components/SaveVideoModal/index.jsx]: Restrict the empty playlist name

**Review by Ashishgupta08 — CHANGES_REQUESTED** [review]

**Comment by Ashishgupta08** [comment]
Remove underline from all the links.
Add more data and create a separate Liked Videos Page.
Check the Play on Hover on Home Page

**Pull request merged**